esp/src/eclwatch/UserQueryWidget.js

```diff
@@ -867,10 +869,32 @@ define([
             registry.byId(this.id + "DeleteGroups").set("disabled", !hasGroupSelection);
             registry.byId(this.id + "ExportGroups").set("disabled", !hasGroupSelection);
 
+
             var permissionSelection = this.permissionsGrid.getSelected();
             var hasPermissionSelection = permissionSelection.length;
-            registry.byId(this.id + "EnableScopeScans").set("disabled", true);
-            registry.byId(this.id + "DisableScopeScans").set("disabled", true);
+            var scopeScansEnabled;
+            var scopeScansDisabled;
+
+            if (hasPermissionSelection && permissionSelection[0].name === "File Scopes") {
+                var context = this;
+                WsAccess.Resources({
+                    request: {
+                        basedn: event.rows[0].id,
+                        rtype: "file",
+                        rtitle: "FileScope"
+                    }
+                }).then(function (response) {
+                    if (lang.exists("ResourcesResponse.scopeScansStatus", response)) {
+                        scopeScansEnabled = response.ResourcesResponse.scopeScansStatus.isEnabled;
+                        scopeScansDisabled = !scopeScansEnabled;
+                        registry.byId(context.id + "EnableScopeScans").set("disabled", scopeScansEnabled);
+                        registry.byId(context.id + "DisableScopeScans").set("disabled", scopeScansDisabled);
+                    } else {
+                        scopeScansEnabled, scopeScansDisabled = true;
```

Comment:
Can you break this down into two lines?  (granted this is because I struggle to remember if this means scopeScansEnabled === undefined or scopeScansEnabled === true).

Reply:
Also setting them here makes no difference as they are not referenced outside of the `then` scope?